src/Tribe/Editor.php

```diff
@@ -211,4 +211,17 @@ public function is_classic_editor() {
 
 		return $is_classic_editor_request || $disabled_by_plugin || $disabled_by_filter;
 	}
+
+	/**
+	 * Returns whether Blocks are being used for Events
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
```

Comment:
```suggestion
	 * @return bool Whether Blocks are being used for Events or not.
```

Add doc comment to all `@return` and `@param` tags.